packages/account-v2/src/constants/errorMessages.tsx

```diff
@@ -0,0 +1,16 @@
+export const API_ERROR_CODES = Object.freeze({
```

Comment:
can we please use instead of Object.freeze 'as const'?
Object.freeze is working just on the top level, but 'as const' makes the object deeply readonly

Reply:
Ok. I wasn't aware of that